Panic for undefined AST element

```
panic: could not match regexp with string
^(?P<address>[0-9a-fx]+) <(?P<position>.*)>[\s]*$
0x20c6ad0 </glibc-2.27/support/temp_file-internal.h:27:62> Inherited


goroutine 13 [running]:
github.com/elliotchance/c2go/ast.groupsFromRegex(0xc4202c0340, 0x31, 0xc4203d90b8, 0x5e, 0x9)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:270 +0x3cc
github.com/elliotchance/c2go/ast.parseWeakAttr(0xc4203d90b8, 0x5e, 0x695e1e)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/weak_attr.go:14 +0x62
github.com/elliotchance/c2go/ast.Parse(0xc4203d90af, 0x67, 0x693ac4, 0x5)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:248 +0xaa7
main.convertLinesToNodes(0xc4201a4af0, 0x390, 0x390, 0x0, 0x0, 0x0)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:94 +0x196
main.convertLinesToNodesParallel.func1.2(0xc4202361e0, 0xc420390000, 0xc4201a4af0, 0x390, 0x390, 0x0)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:126 +0x59
created by main.convertLinesToNodesParallel.func1
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:130 +0x1b6

```

--- a/ast/weak_attr.go
+++ b/ast/weak_attr.go
@@ -4,18 +4,20 @@ package ast
 type WeakAttr struct {
 	Addr       Address
 	Pos        Position
+	Inherited  bool
 	ChildNodes []Node
 }
 
 func parseWeakAttr(line string) *WeakAttr {
 	groups := groupsFromRegex(
-		`<(?P<position>.*)>`,
+		`<(?P<position>.*)>(?P<inherited> Inherited)?`,
 		line,
 	)
 
 	return &WeakAttr{
 		Addr:       ParseAddress(groups["address"]),
 		Pos:        NewPositionFromString(groups["position"]),
+		Inherited:  len(groups["inherited"]) > 0,
 		ChildNodes: []Node{},
 	}
 }

--- a/ast/weak_attr_test.go
+++ b/ast/weak_attr_test.go
@@ -9,6 +9,13 @@ func TestWeakAttr(t *testing.T) {
 		`0x56069ece5110 <line:736:22>`: &WeakAttr{
 			Addr:       0x56069ece5110,
 			Pos:        NewPositionFromString("line:736:22"),
+			Inherited:  false,
+			ChildNodes: []Node{},
+		},
+		`0x20c6ad0 </glibc-2.27/support/temp_file-internal.h:27:62> Inherited`: &WeakAttr{
+			Addr:       0x20c6ad0,
+			Pos:        NewPositionFromString("/glibc-2.27/support/temp_file-internal.h:27:62"),
+			Inherited:  true,
 			ChildNodes: []Node{},
 		},
 	}

